### PR TITLE
Ramya lint fix  for Admin component

### PR DESCRIPTION
**Pull request by ramyaram2092**
# Description
Lint fix for the Admin Component (src/component/Admin)

## Main changes explained:
-  Executed the link checker
-  Fixed changes manually  apart from fix done by lint--fix
-  Removed the component from .prettierignore and .eslintignore

## How to test:
- check into current branch
- do npm install and ... to run this PR locally
- Login as Admin
- go to Other Links->Popup Management
- check the app works as normal 
- set up ESLint and Prettier in VSCode
- check all files in src/component/Admin/** doesn't have errors

### CODE REVIEW

**Comment by ramyaram2092** [comment]
@sxiong5 @AaronPersaud @EvianTan  
Hello everyone .  Since this is my first lint fix I would really appreciate your feedback before releasing this PR for testing

**Review by sxiong5 — APPROVED** [review]
Hey @ramyaram2092, I have tested the PR and everything LGTM!

Screenshot 2023-09-05 at 13 45 51 (2)

Just one little warning I think it's better to remove it.

Screenshot 2023-09-05 at 13 43 07

**Comment by ramyaram2092** [comment]
@sxiong5  Thank you Shihao. I will remove that.

File: .eslintignore [hunk 1]
```diff
@@ -18,8 +18,14 @@ src/styles.js
...
 
+src/jest.config.json
+src/store.js
+src/styles.js
+src/routes.js
+src/registerServiceWorker.js
+src/index.js
 
-src/components/Admin/**
+# src/components/Admin/**
```
Comment (EvianTan) [inline .eslintignore:28]: Please just remove this line, same as in `.prettierigonore` file

File: .eslintignore [hunk 2]
```diff
@@ -18,8 +18,14 @@ src/styles.js
 src/utils/**
 src/__tests__/**
 
+src/jest.config.json
```
Comment (EvianTan) [inline .eslintignore:21]: does this file exist?

File: .eslintignore [hunk 3]
```diff
@@ -18,8 +18,14 @@ src/styles.js
...
+src/store.js
```
Comment (EvianTan) [inline .eslintignore:22]: already in line 16

File: .eslintignore [hunk 4]
```diff
@@ -18,8 +18,14 @@ src/styles.js
...
+src/styles.js
+src/routes.js
+src/registerServiceWorker.js
+src/index.js
```
Comment (EvianTan) [inline .eslintignore:26]: Please remove  line 22-26, already in this file..

File: .prettierignore [hunk 5]
```diff
@@ -16,8 +16,14 @@ src/styles.js
 src/utils/**
 src/__tests__/**
 
-
-src/components/Admin/**
+src/jest.config.json
```
Comment (EvianTan) [inline .prettierignore:19]: please make changes as `.eslintignore`

Comment (ramyaram2092) [reply .prettierignore]: Yes done

File: .eslintignore (hunk 1 above)
Comment (ramyaram2092) [reply .eslintignore]: Done

**Review by EvianTan — APPROVED** [review]

**Pull request merged**